pkg/kubelet/kubelet.go

```diff
@@ -190,10 +193,15 @@ const (
 	eventedPlegMaxStreamRetries = 5
 
 	// backOffPeriod is the period to back off when pod syncing results in an
-	// error. It is also used as the base period for the exponential backoff
-	// container restarts and image pulls.
+	// error.
 	backOffPeriod = time.Second * 10
 
+	// Initial period for the exponential backoff for container restarts.
+	containerBackOffPeriod = time.Second * 10
+
+	// Initial period for the exponential backoff for image pulls.
+	imageBackOffPeriod = time.Second * 10
```

Comment:
thinking out loud. Those backoff objects were already split, correct? So there were no inherited backoff from image backoff? Imagine, we retried images for a few times. Then succeeded and immidiately failed to start container. The backoff will start from the beginning, correct?

Reply:
Yes the backoff objects were already split so in terms of number of retries, or time of a given backoff, they were not dependented on each other.

The purpose of this PR is to separate the value for the starting backoff that those two backoff object are _initialized_ with. It's not obvious here but in a forthcoming PR it will be like:

```
	containerBackOffPeriod = time.Second * 1

	imageBackOffPeriod = time.Second * 10
```

and that's why I need them split.

Reply:
thanks for confirming!